brave/gulpfile.js/common-gulp.js

```diff
@@ -0,0 +1,56 @@
+const gulp = require('gulp')
```

Comment:
This file just copies some of the ./gulpfile.js files. I hope to eventually upstream a change to split them out into another file upstream and then we can just use that.